report2idea/blacklist/blacklist2idea.py

```diff
@@ -105,13 +103,11 @@ def set_common_ip_fields(self):
         self.idea["Source"].append(self.tgt_addr)
 
     def set_common_url_fields(self):
-        self.idea["Note"] = IdeaTemplate.note.format(self.rec["type"].upper(),
-                                                     self.rec["source_url"],
-                                                     self.bl["name"])
-
-        self.idea["Description"] = IdeaTemplate.description_url.format(self.rec["source_url"],
-                                                                       self.bl["name"],
-                                                                       ip_list2description(self.rec["targets"]))
+        self.idea["Description"] = "Suspicious communication with domain name listed on {0} blacklist".format(self.bl["name"])
+        self.idea["Note"] = IdeaTemplate.url_note.format(self.rec["source_url"],
```

Comment:
bad variable name here: url_note -> note_url

Reply:
This was fixed